data/json/vehicle_parts.json

```diff
@@ -3601,7 +3601,7 @@
         "range" : 12,
         "fuel_type" : "plasma",
         "item" : "plasma_rifle",
-        "difficulty" : 7,
+        "difficulty" : 6,
```

Comment:
Power feeds are more exotic than self-contained.  I intended this to apply to all three, and since I missed the plasma engine, please consider that level 8 (on par with minireactor, NOT something taught in the local technical school) as well.